### PR TITLE
Add `recruitment/get-cv` profile `workable` map

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds `workable` map for `recruitment/get-cv` profile.

Based on #333 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: grid/recruitment/get-cv/maps/get-cv.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    describe('GetCV', () => {
+      beforeAll(() => {
+        superface = buildSuperfaceTest({
+          profile: 'recruitment/get-cv',
+          provider,
+          useCase: 'GetCV',
+        });
+      });
+
+      describe('when geting CV with valid candidate ID', () => {
```
Comment (tmladek) [inline grid/recruitment/get-cv/maps/get-cv.ts:27]:
```suggestion
      describe('when getting a CV with a valid candidate ID', () => {
```

File: grid/recruitment/get-cv/maps/get-cv.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        it('returns CV document', async () => {
```
Comment (tmladek) [inline grid/recruitment/get-cv/maps/get-cv.ts:28]:
```suggestion
        it('returns the CV document', async () => {
```

File: grid/recruitment/get-cv/maps/get-cv.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            options
+          );
+
+          expect(() => result.unwrap()).not.toThrow();
+          expect(result).toMatchSnapshot();
+        });
+      });
+
+      testInputs.invalidCandidateIds.forEach(invalidCandidateId => {
+        describe(`when geting CV with invalid candidate ID ${invalidCandidateId}`, () => {
```
Comment (tmladek) [inline grid/recruitment/get-cv/maps/get-cv.ts:45]:
```suggestion
        describe(`when getting a CV with an invalid candidate ID ${invalidCandidateId}`, () => {
```

File: grid/recruitment/get-cv/maps/get-cv.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                input: {
+                  candidateId: invalidCandidateId,
+                },
+                testName: `get CV with invalid candidate ID ${invalidCandidateId}`,
+              },
+              options
+            );
+
+            expect(() => result.unwrap()).toThrow();
+            expect(result).toMatchSnapshot();
```
Comment (tmladek) [inline grid/recruitment/get-cv/maps/get-cv.ts:58]: sanity check - this ensures it really returns `candidate not found` error and not a different error, right?

File: grid/recruitment/get-cv/maps/workable.suma [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    }
+
+    response 200 {       
+      return {}
+    }
+
+    response 422 "application/json" {
+      fail {
+        title = 'Wrong Integration Parameter'
+        detail = `Subdomain ${ args.subdomain } do not exist.`
```
Comment (tmladek) [inline grid/recruitment/get-cv/maps/workable.suma:93]:
```suggestion
        detail = `Subdomain ${ args.subdomain } does not exist.`
```

**Review by tmladek — CHANGES_REQUESTED** [review]
lgtm, except for a couple english fixes

File: grid/recruitment/get-cv/maps/get-cv.ts (hunk 4 above)
Comment (janhalama) [reply grid/recruitment/get-cv/maps/get-cv.ts:58]: yes, the snapshot validation ensures it

**Review by tmladek — APPROVED** [review]
lgtm

**Pull request merged**